AdtSampleApp/SampleClientApp/Program.cs

```diff
@@ -68,28 +57,28 @@ static async Task Main()
                 {
                     client.GetDigitalTwin("---");
```

Comment:
Do we already have the updated Azure.Identity library that would allow us to get rid of this "forcing function" to get auth happen immediately? If so, can we update the sample to use the newest version of the library and remove this?

Reply:
Nope, not yet. Sorry. :(